src/layers/src/geojson-layer/geojson-layer.ts

```diff
@@ -183,15 +193,31 @@ export default class GeoJsonLayer extends Layer {
   declare config: GeoJsonLayerConfig;
   declare visConfigSettings: GeoJsonVisConfigSettings;
   declare meta: GeoJsonLayerMeta;
-  dataToFeature: GeojsonDataMaps;
+
+  dataToFeature: GeojsonDataMaps | BinaryFeatures[];
+  dataContainer: DataContainerInterface | null;
+
+  isArrow: boolean;
```

Comment:
Nit: Move static initializations out of constructor?
```suggestion
  isArrow: boolean = false;
```